WebDriverAgentLib/Routing/FBSession.m

```diff
@@ -178,13 +178,13 @@ - (XCUIApplication *)activeApplication
   if (nil != self.testedApplication) {
     XCUIApplicationState testedAppState = self.testedApplication.state;
     if (testedAppState >= XCUIApplicationStateRunningForeground) {
-      NSPredicate *searchPredicate = [NSPredicate predicateWithFormat:@"%K == %@ OR %K == %@ OR %K == %@",
+      NSPredicate *searchPredicate = [NSPredicate predicateWithFormat:@"%K == %@ OR identifier IN {%@, %@}",
```

Comment:
it is still ok to use %K for identifier

Reply:
Oh, ok. I wondered if `%K` was still valid for `IN`:
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Predicates/Articles/pSyntax.html